spec/support/response_body_shared_context.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.shared_context 'Response Body' do
+  def parsed_response_body
```

Comment:
This is not a `shared_context`! This method is a helper to parse responses, you could use something like this:
```ruby
module Response
  module JSONParser
    def parsed_response_body
      JSON.parse(response.body)
    end
  end
end
```
Check if this is being included globally, if not you can `include Response::JSONParser` in the files where you need it.